flask_expects_json/__init__.py

```diff
@@ -44,6 +44,10 @@ def decorated_function(*args, **kwargs):
                 return abort(400, e)
 
             g.data = data
-            return f(*args, **kwargs)
+
+            if hasattr(current_app, "ensure_sync"):
```

Comment:
@Fischerfredl this if statement here should ensure the library is still compatible with old version of flask without async support